.gitignore

```diff
@@ -1,13 +1,42 @@
-*.class
+# copied from https://github.com/Open-RIO/.gitignore/blob/master/.gitignore
 
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
+# WPILib
+bin/
+build/
+dist/
+sysProps.xml
+workbench.xmi
 
-# Package Files #
-*.jar
```

Comment:
Are we sure that we don't want to ignore .jar files?

Reply:
We were previously versioning the CTRE jars in `/lib` but that may be an anti-pattern. They were removed from git in #6 by @BearsAreAwsome when the gitignore file was first updated. I _think_ we ignore the jars created as build artifacts by ignoring the contents of `build/` and `dist/`, but honestly don't know enough about java build outputs to be sure. Want to re-add the ignore line for `*.[ejw]ar`?

Reply:
I agree we should NOT be versioning the CTRE or any other jars. Assuming for the build artifacts by ignoring `build` and `dist` works for me. The only reason I'd explicitly add `.jar` back would be to prevent accidentally committing any libs.